icicle/tests/test_hash_api.cpp

```diff
@@ -27,15 +28,12 @@ class HashApiTest : public IcicleTestBase
   template <typename T>
   static void randomize(T* arr, uint64_t size)
   {
-    // Create a random number generator
-    std::random_device rd;                                       // Non-deterministic random number generator
-    std::mt19937 gen(rd());                                      // Mersenne Twister engine seeded with rd()
     std::uniform_int_distribution<uint32_t> dist(0, UINT32_MAX); // Range of random numbers
 
     // Fill the array with random values
     uint32_t* u32_arr = (uint32_t*)arr;
     for (int i = 0; i < (size * sizeof(T) / sizeof(uint32_t)); ++i) {
-      u32_arr[i] = dist(gen);
+      u32_arr[i] = dist(rand_generator);
```

Comment:
why not use rand_int()?

Reply:
Because I changed this part before noticing all the rand() instances that needed to be addressed
I'll swap to rand_int